event-sourcing/event-source-api-integration-test/src/test/java/uk/gov/justice/services/eventsourcing/source/api/EventStreamsFeedIT.java

```diff
@@ -112,24 +110,24 @@ public WebApp war() {
 
     @Before
     public void setUp() throws Exception {
-        eventStreamsFeedService.initialiseWithPageSize(25);
+        eventStreamsFeedService.initialiseWithPageSize();
```

Comment:
think this method is now redundant, so just call initialise? and shouldn't be needed in sub methods and they are not changing page sizes.

Reply:
Refactored